LanguageFeatures/primary-constructors/static_processing_A06_t02.dart

```diff
@@ -29,56 +29,64 @@
 
 import '../../Utils/expect.dart';
 
-extension type ET1(String x) {
-  String Function() captureAtDeclaration = () => x;
-  String Function() captureInInitializer;
+String log = "";
 
-  this : captureInInitializer = (() => x);
+extension type ET1(String x) {
+  this : assert("$log=$x".isNotEmpty);
```

Comment:
Perhaps?:
```suggestion
  this : assert("${log = x}".isNotEmpty);
```
I'm assuming we want the assignment to be executed.

Reply:
Yes, a typo. Thank you. Fixed here and below.